tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "strict": true,
```

Comment:
To compile with `"strict": true`, we'll need to stricten all the tests too. That's too much to do in this PR, so for now, let's remove this line.

Reply:
Done.